app_grim_reefer/boards/arm/grim_reefer/grim_reefer_defconfig

```diff
@@ -0,0 +1,68 @@
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_STM32F4X=y
+CONFIG_SOC_STM32F446XX=y
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable HW stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# enable pin controller
+CONFIG_PINCTRL=y
+
+# ADC
+CONFIG_ADC=y
+CONFIG_ADC_ASYNC=y
+
+# Clocks
+CONFIG_CLOCK_CONTROL=y
+
+# Console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SHELL=y
+
+# GPIO
+CONFIG_GPIO=y
+CONFIG_LED=y
+
+# IO
+CONFIG_GPIO=y
+CONFIG_SERIAL=y
+
+CONFIG_SPI=y
+CONFIG_SPI_ASYNC=y
+#CONFIG_SPI_RTIO=y
+
+CONFIG_I2C=y
+CONFIG_I2C_CALLBACK=y
+#CONFIG_I2C_RTIO=y
+
+CONFIG_SENSOR_ASYNC_API=y
+CONFIG_SENSOR=y
+
+# Random
+CONFIG_TEST_RANDOM_GENERATOR=y
+
+# Filesystem
+CONFIG_FLASH=y
+CONFIG_FLASH_MAP=y
+CONFIG_FLASH_PAGE_LAYOUT=y
+CONFIG_DISK_ACCESS=y
+CONFIG_FILE_SYSTEM=y
+CONFIG_FILE_SYSTEM_LITTLEFS=y
+CONFIG_FS_LITTLEFS_BLK_DEV=y
+
+
+# Networking
+CONFIG_NETWORKING=y
+CONFIG_NET_IPV4=y
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_NET_L2_ETHERNET_MGMT=y
+CONFIG_NET_STATISTICS=y
+CONFIG_NET_DEFAULT_IF_ETHERNET=y
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
+CONFIG_PTP_CLOCK=y
```

Comment:
```suggestion
```

Reply:
fixed in [6cf8032](https://github.com/RIT-Launch-Initiative/FSW/pull/52/commits/6cf803229d38690de83d6db332228645b0495534)